ui/apps/platform/src/Components/TableStateTemplates/TbodyError.tsx

```diff
@@ -10,24 +10,26 @@ export type TbodyErrorProps = {
     colSpan: number;
     error: unknown;
     headingLevel?: EmptyStateTemplateProps['headingLevel'];
+    title?: string;
     message?: string;
 };
 
 export function TbodyError({
     colSpan,
     error,
     headingLevel = 'h2',
-    message = 'An error has occurred. Try clearing any filters or refreshing the page',
+    title = 'An error has occurred. Try clearing any filters or refreshing the page',
+    message,
 }: TbodyErrorProps) {
     return (
         <TbodyFullCentered colSpan={colSpan}>
             <EmptyStateTemplate
                 headingLevel={headingLevel}
-                title={getAxiosErrorMessage(error)}
+                title={title}
                 icon={ExclamationCircleIcon}
                 iconClassName="pf-v5-u-danger-color-100"
             >
-                {message}
+                {message ?? getAxiosErrorMessage(error)}
             </EmptyStateTemplate>
         </TbodyFullCentered>
```

Comment:
In cases where we want to display the network error message, displaying it in the body is more appropriate than in the title.